src/Mod/Sketcher/App/planegcs/GCS.cpp

```diff
@@ -5405,7 +5405,7 @@ void System::identifyDependentParametersSparseQR(const Eigen::MatrixXd& J,
                               nontransprank,
                               Rparams,
                               false,
-                              true);  // do not transpose allow to diagnose parameters
+                              true);  // do not transpose allow one to diagnose parameters
```

Comment:
This comment is sort of non-sensical (not your fault, obviously). I wonder if it's two comments, one about the first boolean and one about the second. @PaddleStroke or @AjinkyaDahale do you know?